sdk/storage/perf-test/pom.xml

```diff
@@ -18,12 +19,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.0.0</version>
+      <version>12.4.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-perfstress</artifactId>
-      <version>1.0-SNAPSHOT</version>
+      <artifactId>perf-test-core</artifactId>
+      <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:perf-test-core;dependency} -->
```

Comment:
This is a new dependency that is built outside of the storage pipeline. There needs to be a module entry for this dependency in the sdk/storage/pom.service.xml